bazel/repositories.bzl

```diff
@@ -197,17 +197,18 @@ def proxy_wasm_cpp_host_repositories():
         http_archive,
         name = "com_github_bytecodealliance_wasm_micro_runtime",
         build_file = "@proxy_wasm_cpp_host//bazel/external:wamr.BUILD",
-        # WAMR-1.2.1
-        sha256 = "7548d4bbea8dbb9b005e83bd571f93a12fb3f0b5e87a8b0130f004dd92df4b0b",
-        strip_prefix = "wasm-micro-runtime-WAMR-1.2.1",
-        url = "https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-1.2.1.zip",
+        # WAMR-2.1.1
+        sha256 = "a0824762abbcbb3dd6b7bb07530f198ece5d792a12a879bc2a99100590fdb151",
+        strip_prefix = "wasm-micro-runtime-WAMR-2.1.1",
+        url = "https://github.com/bytecodealliance/wasm-micro-runtime/archive/refs/tags/WAMR-2.1.1.zip",
     )
 
     native.bind(
         name = "wamr",
         actual = "@com_github_bytecodealliance_wasm_micro_runtime//:wamr_lib",
     )
 
+    #TODO: switch to the system pre-installed building. still 15.0.7 for sure
```

Comment:
We want to make builds even more hermetic, and relying on a system-installed LLVM is definitely not the way to do it. I'd remove this comment.

Reply:
good to know